README.md

```diff
@@ -104,6 +105,14 @@ The AWS IoT Device Defender library has no dependencies on additional libraries
 
 See memory requirements for the latest release [here](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/device-defender-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#defender_memory_requirements).
 
+#### AWS IoT Over-the-air Updates
```

Comment:
Add "(Release Candidate)"

Reply:
Addressed in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1454/commits/aa538749dedbcaddc4d3c7b0179ef4d1ce21392d